osu.Game.Tests/Visual/Gameplay/TestSceneSpectator.cs

```diff
@@ -335,12 +336,10 @@ private void loadSpectatingScreen()
         private class DependenciesScreen : OsuScreen
         {
             [Cached(typeof(SpectatorClient))]
-            public readonly TestSpectatorClient Client;
+            public readonly TestSpectatorClient Client = new TestSpectatorClient();
```

Comment:
Can this be named `SpectatorClient`? I was really confused for a second which client this was and thought it was accidentally removed when comparing with the old version.

Reply:
I've updated this and a few other ambiguous `Client` fields.